lib/network_utils.ml

```diff
@@ -921,8 +922,31 @@ module Ovs = struct
 			| Some x, None -> ["--"; "set"; "bridge"; name; "mcast_snooping_enable=" ^ (string_of_bool x)]
 			| _ -> []
 		in
-		vsctl ~log:true (del_old_arg @ ["--"; "--may-exist"; "add-br"; name] @
-			vlan_arg @ mac_arg @ fail_mode_arg @ disable_in_band_arg @ external_id_arg @ vif_arg @ set_mac_table_size @ set_igmp_snooping)
+        let igmp_snooping_old =
```

Comment:
This deserves to be a separate function to get the current state for IGMP snooping.